src/ipc-handlers.ts

```diff
@@ -579,17 +595,34 @@ export async function getThumbnailData( _event: IpcMainInvokeEvent, id: string )
 export function openTerminalAtPath( _event: IpcMainInvokeEvent, targetPath: string ) {
 	return new Promise< void >( ( resolve, reject ) => {
 		const platform = process.platform;
+		const cliPath = nodePath.join( getResourcesPath(), 'bin' );
+
+		const exePath = app.getPath( 'exe' );
+		const appDirectory = app.getAppPath();
+		const appPath =
+			process.env.NODE_ENV === 'development' ? `${ exePath } ${ appDirectory }` : exePath;
```

Comment:
I spotted that this condition should be `! app.isPackaged` because the Electron shell app is invoked when the app is not packaged, i.e. produces its own executable. I'll address it in a separate PR.

Reply:
This is addressed in https://github.com/Automattic/studio/pull/268.